ksql-engine/src/main/java/io/confluent/ksql/metrics/MetricCollectors.java

```diff
@@ -16,16 +16,15 @@
 
 package io.confluent.ksql.metrics;
 
+import io.confluent.common.utils.Time;
 import org.apache.kafka.common.metrics.JmxReporter;
 import org.apache.kafka.common.metrics.MetricConfig;
 import org.apache.kafka.common.metrics.Metrics;
 import org.apache.kafka.common.metrics.MetricsReporter;
 import org.apache.kafka.common.utils.SystemTime;
 
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
import * 
you should change your ide settings

Reply:
done